Jenkinsfile_nightly

```diff
@@ -33,6 +33,13 @@ def integrationTestSecrets = [
   ]
 ]
 
+def e2eTestSecrets = [
+  'fpl-${env}': [
```

Comment:
wonder if this needs to be fpl-aat, looking at the integrationTestSecrets above it looks like the env is hardcoded?
```suggestion
  'fpl-aat': [
```